docs/commands.md

```diff
@@ -406,6 +421,34 @@ This section contains information on all supported functions from the Cypher que
 |id() | Returns the ID of a relationship or node |
 |labels() | Returns a string representations of the label of a node. |
 
+## Indexing
+RedisGraph supports single-property indexes for node labels.
+The creation syntax, to be provided as a GRAPH.QUERY command, is:
+
+```sh
+CREATE INDEX ON :person(age)
```

Comment:
This might be a bit confusing, would we prefer GRAPH.QUERY <GRAPH_ID> "CREATE INDEX ON :person(age)" ?